packages/slate-editor/src/components/EditorBlock/EditorBlock.tsx

```diff
@@ -203,6 +203,7 @@ export const EditorBlock = forwardRef<HTMLDivElement, Props>(function (
                         [styles.border]: border,
                         [styles.editable]: Boolean(renderEditableFrame),
                         [styles.hasError]: hasError,
+                        [styles.overflowAuto]: overflow === 'auto',
```

Comment:
I was wondering if we need these classes or would it be simpler to just pass the `overflow` to the `style` property? 🤔 

We're using `style` in the `ParagraphElement` for alignment for example, so I felt like we could use it here as well since we're using the `overflow` property as-is.

Reply:
Makes sense :)